src/core/cfg.cpp

```diff
@@ -1932,6 +1932,11 @@ class CFGVisitor : public ASTVisitor {
         return true;
     }
 
+    bool visit_ellipsis(AST_Ellipsis* node) override {
+        // TODO:
+        return true;
+    }
```

Comment:
My recollection of the cfg code isn't great, but I think Ellipsis should be handled in remapExpr rather than as a top-level visit function (I'm surprised you didn't have to add that).